_docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md

```diff
@@ -0,0 +1,70 @@
+
+## Parameters
+
+- `templateId` references the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) received from its creator.
+- Optionally, `creationContent` can be provided. It is a [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent) depending on whether the RelationshipTemplate's `content` is a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) or an [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent). Specify this parameter if additional validations are wanted for the `creationContent` that would be used to create the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship).
+
+## On Success
+
+- Returns a CanCreateRelationshipResponse that indicates if a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) can be created with the given parameters.
+- If the `isSuccess` property of the CanCreateRelationshipResponse has the value `true`, a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) can currently be created to the creator of the RelationshipTemplate. This would then initially have `"Pending"` as `status` until the creator of the RelationshipTemplate [accepts the Relationship]({% link _docs_use-cases/use-case-transport-accept-relationship.md %}).
+- If the `isSuccess` property of the CanCreateRelationshipResponse has the value `false`, no [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) can currently be created to the creator of the RelationshipTemplate. This can have various causes, for example:<br>
+  -- The `templateId` does not resolve to a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) or the associated RelationshipTemplate was not cached correctly.<br>
+  -- A [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with `"Pending"`, `"Active"`, `"Terminated"` or `"DeletionProposed"` as `status` already exists to the creator of the RelationshipTemplate.<br>
+  -- The [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) has already expired, which means that the timestamp specified in its `expiresAt` property has been exceeded.<br>
+  -- The potential initiator has already [decomposed the former Relationship]({% link _docs_integrate/terminate-relationships.md %}#decompose-a-relationship) to the creator of the RelationshipTemplate, but the RelationshipTemplate's creator has not yet decomposed it.<br>
```

Comment:
This is the case if a Relationship with `"DeletionProposed"` as `status` already exists to the creator of the RelationshipTemplate, which you mentioned in the second point. I do think it is good to point this out explicitly again, since I don't expect the average integrator to know this detail immediately. However, I'd either move this point directly below it or even combine it with the other point by adding another sentence with more details how a Relationship with `"DeletionProposed"` as `status` can exist (without being visible).